schedule/virt_autotest/tumbleweed-virt-guest-install.yaml

```diff
@@ -17,6 +15,12 @@ conditional_schedule:
         DO_NOT_INSTALL_HOST:
```

Comment:
if `DO_NOT_INSTALL_HOST` is 1, it will directly go to {{install_guest}}? Miss login_console?

Reply:
Yes, `DO_NOT_INSTALL_HOST` is meant to run guest installation test or feature tests on an installed system, so no restart or login is required, but you have to make sure the machine is in running state and to trigger test with `IPMI_DO_NOT_RESTART_HOST=1`. It is helpful for debugging.